marked-element.html

```diff
@@ -149,17 +158,35 @@
         observer: 'render',
         type: Function,
         value: null
+      },
+      /**
+       * A reference to the XMLHttpRequest instance used to generate the
+       * network request.
+       *
+       * @type {XMLHttpRequest}
+       */
+      xhr: {
+        type: Object,
+        notify: true,
+        readOnly: true,
+        value: function() {
+          return new XMLHttpRequest();
+        }
       }
     },
 
     ready: function() {
-      if (!this.markdown) {
-        // Use the Markdown from the first `<script>` descendant whose MIME type starts with
-        // "text/markdown". Script elements beyond the first are ignored.
-        var markdownElement = Polymer.dom(this).querySelector('[type^="text/markdown"]');
-        if (markdownElement != null) {
-          this.markdown = this._unindent(markdownElement.textContent);
-        }
+      if (this.markdown) return;
+
+      // Use the Markdown from the first `<script>` descendant whose MIME type starts with
+      // "text/markdown". Script elements beyond the first are ignored.
+      var markdownElement = Polymer.dom(this).querySelector('[type^="text/markdown"]');
+      if (!markdownElement) return;
+
+      if (markdownElement.src) {
+        this._request(markdownElement.src);
+      } else {
+        this.markdown = this._unindent(markdownElement.textContent);
```

Comment:
Could we support placeholder text while the request is in progress? We could use the script contents as the placeholder, overwritten when the ajax request comes back. 

Something like this:

```
<marked-element>
  <div class="markdown-html"></div>
  <script type="text/markdown" src="../guidelines.md">
    Loading markdown...
  </script>
</marked-element>
```